File: joseffe-barroso/Java/src/entrada_processamento_saida/aula03/ddd/Exercise29.java
package ddd;

import java.util.Arrays;
import java.util.Scanner;
import java.util.Collections;

public class Exercise29 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		int firstNumber, secondNumber, thirdNumber;

		System.out.print("Digite o primeiro número: ");
		firstNumber = scanner.nextInt();

		System.out.print("Digite o segundo número: ");
		secondNumber = scanner.nextInt();

		System.out.print("Digite o terceiro número: ");
		thirdNumber = scanner.nextInt();

		Integer[] listNumbers = { firstNumber, secondNumber, thirdNumber };

		Arrays.sort(listNumbers, Collections.reverseOrder());
		System.out.println("\nOrdem crescente dos valores: " + Arrays.toString(listNumbers));

		scanner.close();
	}

}

File: joseffe-barroso/Python/aula03/exercise29.py
firstNumber = int(input('Digite o primeiro número: '))
secondNumber = int(input('Digite o segundo número: '))
thirdNumber = int(input('Digite o terceiro número: '))

listNumbers = [firstNumber, secondNumber, thirdNumber];

print("Ordem decrescente dos valores", sorted(listNumbers, reverse=True))